### DISCUSSION TITLE
(Ruby) Heredoc: While heredoc is not complete there is a Ghost error always

### DISCUSSION THREAD

**Discussion by zealot128** [nvim-treesitter/nvim-treesitter#2101] [Q&A] [answered]
When inputting a Heredoc Comment, there is always a ghost comment in the end:

https://user-images.githubusercontent.com/147175/145847047-8de879f2-011f-44cd-8a2e-9770e78aee21.mp4

Does anybody else have the same problem? Not sure if it is a issue just with my configuration.

Tried upgrading Treesitter to HEAD + ``:TSInstall ruby`` on a Neovim 0.6

Maybe reason? TSPlayground shows a "heredoc_end" but there is none yet:
![Bildschirmfoto 2021-12-13 um 17 10 00](https://user-images.githubusercontent.com/147175/145847270-53a15a92-025e-4395-97fd-534b7dabec7d.png)

Config:

```lua
ts_config.setup {
     ensure_installed = {
        "lua",
     },
     highlight = {
        enable = true,
        use_languagetree = true, -- tried setting false/true -> same error?
        additional_vim_regex_highlighting = false,
     },
     incremental_selection = {
        enable = true,
     },
     matchup = {
        enable = false,
     },
     context_commentstring = {
       enable = true
     }
  }
```

**Comment by theHamsta** [comment, marked as answer]
~~Can you get the line number it's showing? It's not visible (cut off) in the video?~~ It's query.lua:181

**Reply by theHamsta** [reply]
Which nvim version are you using? On my build 181 is a empty line. It seems to fail quite near `get_node_text`.

Also there should be an exact error description when you manage to make your terminal wider.

**Reply by zealot128** [reply]
used ``neovim-0.6.0~ubuntu1+git202111301248-04c7b5503-adeb5640f``

Upgraded to:

```
0.6.0~ubuntu1+git202112131249-9a0196d24-adeb5640f-bb94ef4f0~ubuntu18.04.1
```

Now it works! Thanks for the quick help. Should upgrade neovim head more often :)
